README.md

```diff
@@ -33,7 +33,7 @@
 
 ## Release notes
 
-### Version `3.0.0`
```

Comment:
What happen to 3.0.0 version? Why the changelog now belongs to 3.0.1.pre-1? (btw, it seems that it is commonly used `3.0.1-per.1` over `3.0.1.pre-1`)

Reply:
Oh, that's a typo

Reply:
Should we just remove the git tag? :thinking: